script.js

```diff
@@ -1,28 +1,86 @@
-var values  = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A"];
+var values  = [2, 3, 4, 5, 6, 7, 8, 9, 10, "Jack", "Queen", "King", "Ace"];
 var suits   = ["Clubs", "Diamonds", "Hearts", "Spades"];
 var game = {
   deck: [],
   players: [],
   hands: [],
   buildDeck: function(){
-
+      for (var myRank = 0; myRank < values.length; myRank++) {
+        for (var mySuit = 0; mySuit < suits.length; mySuit++) {
+          this.deck.push({ "rank" : values[myRank], "suit" : suits[mySuit]});
+        }
+      }
   },
   shuffleDeck: function(){
+    var currentIndex = this.deck.length, temporaryValue, randomIndex;
+
```

Comment:
You're absolutely right, and I meant to do that. I used the exact same code for project 1, and I did mention I copied it from Stack Overflow.